geometry_msgs/Point[] points
std_msgs/Int32[] labels
int32 obj_id
bool clear_old_points
bool reset
